src/Molecules/Filter/Filter.jsx

```diff
@@ -1,8 +1,10 @@
 import React, { useState } from 'react'
 
+import { Filter as FilterIcon } from 'styled-icons/boxicons-regular/Filter'
 import { Trash } from 'styled-icons/boxicons-solid/Trash'
 import { Check } from 'styled-icons/material/Check'
 import Divider from '@material-ui/core/Divider'
+import Popover from '@material-ui/core/Popover'

```

Comment:
Can we use our Popover instead of material-ui's ? We can then have *beautiful* bootstrap styles

Reply:
I used the popover that was already used in the SortedColumns, it's not part of this story to migrate from it (and I doubt it will ever be).